apps/web/src/serlo-editor-integration/create-plugins.tsx

```diff
@@ -94,13 +107,13 @@ export function createPlugins({
     },
     {
       type: EditorPluginType.Spoiler,
-      plugin: createSpoilerPlugin(),
+      plugin: createSpoilerPlugin({}), // Passing empty config object here allows all plugins
       visibleInSuggestions: true,
       icon: <IconSpoiler />,
     },
     {
       type: EditorPluginType.Box,
-      plugin: createBoxPlugin({}),
+      plugin: createBoxPlugin(),
```

Comment:
Can you please explain why you passed the empty object to the spoiler, but removed it from the box?

Reply:
When I changed the default `allowedPlugins` for the integrations that messed with the `allowedPlugins` for serlo.org. This is a quick (but very unclear) solution to restore the original nesting restrictions on serlo.org without changing more things.